tool/rig/main.go

```diff
@@ -197,6 +197,11 @@ func freeze(ctx context.Context, client *kubernetes.Clientset, config *rest.Conf
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	committed, _ := cs.IsCommitted(ctx, namespace, changeset.Name)
+	if committed {
+		// Nothing to do
```

Comment:
I would at least print something to stdout to indicate this is the case. Same below.